auto-dark.el

```diff
@@ -26,6 +26,8 @@
 
 
 ;;; Code:
+(require 'dbus)
```

Comment:
Hello there! Thanks for your prompt response!

Indeed this solves the problem, but I think a top general require for something that is not "core" for all OSs might be a bit off (stylistic speaking).

Let me elaborate a little :)

I was thinking on "isolate" this require to the `gnu/linux` portion of the code. Specifically on the entry function here: https://github.com/LionyxML/auto-dark-emacs/blob/master/auto-dark.el#L78

Maybe:
```elisp
(defun auto-dark--is-dark-mode-dbus ()
  "Use Emacs built-in D-Bus function to determine if dark theme is enabled."
  (when (require 'dbus nil t)
    (eq 1 (caar (dbus-call-method
		 :session
		 "org.freedesktop.portal.Desktop"
		 "/org/freedesktop/portal/desktop"
                 "org.freedesktop.portal.Settings" "Read"
                 "org.freedesktop.appearance" "color-scheme")))))
``` 
This would:
- Require the dbus package only on a condition it might be present (gnu/linux)
- Stop the code execution if the package is not present and free us of dealing with some exception specific for a user case that have a window manager / system configured in a non conventional "d-bus way".

How do you feel about this approach?